src/library_eventloop.js

```diff
@@ -73,6 +73,7 @@ LibraryJSEventLoop = {
     '$runtimeKeepalivePush', '$runtimeKeepalivePop',
 #endif
   ],
+  emscripten_set_immediate__sig: 'iii',
```

Comment:
Can you use `p` for pointers of things of size_t size?

So this would be `ipp`